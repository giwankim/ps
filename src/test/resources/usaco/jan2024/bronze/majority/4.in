10
50
19 42 42 19 36 9 19 39 38 19 48 48 19 20 20 19 43 43 19 39 34 19 39 44 19 34 34 19 42 42 19 15 3 19 21 9 19 25 25 19 36 36 19 40 2 19 42 42 19 34
50
35 17 12 46 5 40 32 41 10 44 33 22 43 47 2 27 25 8 50 23 25 19 18 5 33 22 34 27 6 41 48 2 49 40 28 41 50 46 37 27 19 28 42 49 10 47 8 20 26 47
50
4 45 24 39 45 49 46 45 10 28 45 38 41 45 14 14 45 1 35 45 1 1 45 10 10 45 46 32 45 34 34 45 35 43 45 39 14 45 18 18 45 4 12 45 42 42 45 37 37 45
50
46 41 43 41 12 16 24 36 24 47 44 35 15 18 12 50 21 34 29 3 18 41 49 28 29 10 36 38 33 1 44 15 12 27 30 11 29 37 26 11 29 45 42 50 16 15 5 17 11 45
50
37 16 16 37 1 1 37 19 19 37 30 47 37 17 28 37 47 18 37 15 15 37 50 50 37 20 20 37 10 5 37 24 1 37 26 26 37 20 20 37 14 49 37 49 19 37 23 11 37 48
50
37 22 19 17 24 16 13 40 9 24 45 19 8 16 16 17 23 9 15 34 3 15 44 4 21 5 29 41 30 29 5 4 46 24 38 31 14 17 29 42 7 44 42 19 31 14 42 17 42 45
50
30 16 7 7 16 27 25 16 9 23 16 1 1 16 44 44 16 28 8 16 37 37 16 10 10 16 19 27 16 33 33 16 48 48 16 19 17 16 21 21 16 22 33 16 4 4 16 43 21 16
50
5 11 10 26 12 21 4 50 26 43 8 46 13 6 28 18 5 29 13 37 43 47 9 33 40 32 27 10 21 19 25 38 22 1 49 9 9 17 10 34 33 47 16 11 12 26 10 43 7 34
50
22 2 32 22 8 42 22 45 45 22 33 33 22 40 40 22 32 32 22 34 34 22 2 27 22 2 20 22 7 7 22 7 26 22 28 28 22 5 5 22 41 41 22 26 12 22 37 12 22 46
50
15 34 24 33 30 1 35 7 50 48 10 44 26 4 44 39 18 32 12 47 25 7 44 14 50 11 6 17 3 1 22 9 40 42 43 20 3 20 35 8 7 37 29 43 24 49 24 7 36 6
