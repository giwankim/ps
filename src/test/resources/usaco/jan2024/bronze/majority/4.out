20 25 34 36 42 43 48
-1
1 10 14 18 34 37 42
24 41
1 15 16 19 20 26 49 50
16 42
1 4 7 10 21 33 37 44 48
9
5 7 28 32 33 34 40 41 45
20 24
